components/jss/run-priv-sock-server.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
-STATEDIR=${STATEDIR:-/host/tsi-secure}
+SOCKETFILE="/host/sockets/app.sock"
 cd /usr/local/bin || exit
-uwsgi --http-socket /host/sockets/app.sock --chmod-socket=666 --manage-script-name --mount /=web-server-priv:app --plugins python 
+uwsgi --http-socket ${SOCKETFILE} --chmod-socket=666 --manage-script-name --mount /=web-server-priv:app --plugins python &
+echo "wait for the socket file to be created then change its security context..."
+sleep 15
```

Comment:
Hmm, how about waiting on file exists, like 
```while [ ! -f /tmp/x/y/z ]; do sleep 5; done```

same for other sleep waiting for sockets.

Reply:
fixed